alexandria/core/management/commands/generate_content_vectors.py

```diff
@@ -17,14 +17,20 @@ def handle(self, *args, **options):
                 )
             )
 
+        # disable additional features to focus on vectorization
+        checksum = settings.ALEXANDRIA_ENABLE_CHECKSUM
+        thumbnail = settings.ALEXANDRIA_ENABLE_THUMBNAIL_GENERATION
```

Comment:
I don't think you need to keep these, as it's changed in the management command's state only. No need to change it back later

Reply:
Removed